anchore_engine/services/policy_engine/api/controllers/feeds.py

```diff
@@ -205,7 +224,7 @@ def delete_feed(feed):
         session.rollback()
 
     try:
-        f = sync.DataFeeds.delete_feed(feed)
+        f = sync.DataFeeds.delete_feed(internal_feed_name)
```

Comment:
Ditto (although I know you didn't add this(.

Reply:
done